db/migrate/20230505011116_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :author_id
+      t.string :text
```

Comment:
- [ ] For the comment text, a `string` column would be insufficent. `:string` is mapped to `VARCHAR` and has a limit of 255 characters. 
As a general rule of thumb, use `:string` for short text input (username, email, password, titles, etc.) and use `:text` for longer expected input such as descriptions, post body, comment content etc.